code/modules/surgery/organs/_organ.dm

```diff
@@ -320,7 +320,7 @@ INITIALIZE_IMMEDIATE(/obj/item/organ)
 	// Default organ fixing handling
 	// May result in kinda cursed stuff for mobs which don't need these organs
 	var/obj/item/organ/internal/lungs/lungs = get_organ_slot(ORGAN_SLOT_LUNGS)
-	if(!lungs)
+	if(!lungs && !HAS_TRAIT(src, TRAIT_NOBREATH))
```

Comment:
```suggestion
	if(!lungs && !HAS_TRAIT_FROM(src, TRAIT_NOBREATH, SPECIES_TRAIT)
```

Reply:
I didn't specify the source specifically because it won't come from the species, because if it did, it would be caught above. It's for if there was a case where the lungs got removed and the trait was added, and we typically won't want them to be given back in that case.

Reply:
okay but if they're infected with, like, self-resp or are wearing a xenobio rebreather, they'll die from having no lungs once they lose either of those, no?